frontend/src/pages/account/Security/dialogs/TokenDialog.vue

```diff
@@ -0,0 +1,137 @@
+<template>
+    <ff-dialog ref="dialog" data-el="add-token-dialog" header="Add Token" :confirm-label="token ? 'Save' : 'Create'" :disablePrimary="disableConfirm" @confirm="confirm()">
```

Comment:
```suggestion
    <ff-dialog ref="dialog" data-el="add-token-dialog" :header="(token ? 'Edit' : 'Add') + ' Token'" :confirm-label="token ? 'Save' : 'Create'" :disablePrimary="disableConfirm" @confirm="confirm()">
```

Update title to reflect add/edit mode